### PR TITLE
Myronkaifung data out and admin

### PR DESCRIPTION
**Pull request by myronkaifung**

### CODE REVIEW

**Comment by myronkaifung** [comment]
tldr:

1. New Login Methods page
2. New Export Methods page
3.  Directory pages for Integrations
4. Standardized naming for Pipelines to align with dev docs
5. Clean up ToC

File: pages/docs/access-security/single-sign-on/overview.mdx
Comment (Tofufu) [inline pages/docs/access-security/single-sign-on/overview.mdx:1]: quick typo - should be whose: The SCIM endpoint only affects users whose email has a domain in the list of your verified claimed domains.

File: pages/docs/cohort-sync/integrations.mdx
Comment (Tofufu) [inline pages/docs/cohort-sync/integrations.mdx:1]: airship is on here 2x

Comment (myronkaifung) [reply pages/docs/cohort-sync/integrations.mdx:1]: fixed

File: pages/docs/access-security/single-sign-on/overview.mdx
Comment (myronkaifung) [reply pages/docs/access-security/single-sign-on/overview.mdx:1]: fixed

**Review by Tofufu — APPROVED** [review]
lgtm!

**Pull request merged**